pulp_smash/utils.py

```diff
@@ -108,14 +108,19 @@ def reset_pulp(server_config):
     # Why not use runuser's `-u` flag? Because RHEL 6 ships an old version of
     # runuser that doesn't support the flag, and RHEL 6 is a supported Pulp
     # platform.
-    client = cli.Client(server_config)
-    prefix = '' if is_root(server_config) else 'sudo '
+    system = server_config.get_systems('mongod')[0]
+    client = cli.Client(server_config, pulp_system=system)
     client.run('mongo pulp_database --eval db.dropDatabase()'.split())
-    client.run((
-        prefix + 'runuser --shell /bin/sh apache --command pulp-manage-db'
-    ).split())
-    client.run((prefix + 'rm -rf /var/lib/pulp/content').split())
-    client.run((prefix + 'rm -rf /var/lib/pulp/published').split())
+
+    for index, system in enumerate(server_config.get_systems('api')):
+        prefix = '' if is_root(server_config, pulp_system=system) else 'sudo '
+        if index == 0:
```

Comment:
Is the intent to execute `pulp-manage-db` on the same system hosting mongod? If so, I don't  think this will work. There's no guarantee that `server_config.get_systems('mongod')[0]` and `server_config.get_systems('api')[0]` are the same system. Can we use the same `client` object to execute both `db.dropDatabase()` and `pulp-manage-db`?